sycl/test/esimd/on-device/BitonicSortK.cpp

```diff
@@ -6,8 +6,8 @@
 //
 //===----------------------------------------------------------------------===//
 // TODO enable on Windows
-// REQUIRES: linux
-// REQUIRES: gpu
+// REQUIRES: linux && gpu
+// UNSUPPORTED: cuda
```

Comment:
I still think // UNSUPPORTED: cuda || level_zero should be in all tests which can run on opencl BE only

Reply:
I think this will effectively disable ESIMD tests in CI, where machines have either CUDA or LevelZero + OpenCL devices.

Reply:
@vladimirlaz, @bader - I will wait for your decision on the best variant then